doc/adoc/user_guide.adoc

```diff
@@ -62,9 +62,62 @@ To install the distribution migration system run:
 [listing]
 tux > sudo zypper in SLE15-Migration suse-migration-sle15-activation
 
+== Customize the Migration Process
+
+The migration system reads a custom config file from the system to upgrade.
+The contents of that file allows to influence the behavior of the upgrade
+process. Prior to the start of the upgrade process, create the following
+file:
+
+[listing]
+tux > ssh INSTANCE_USER@IP_OF_INSTANCE 'touch /etc/sle-migration-service.yml'
+
+The custom config file allows for the following settings:
+
+Enable Debug Mode::
+If enabled, prevents the upgrade system from rewinding the setup
+steps and rebooting due to a failed upgrade, allowing the issue to
+be debugged.
+
+[listing]
+debug: true|false
+
+Specify Migration Product::
+With the SLE15-Migration package a predefined product setup is
+already applied. The value for the target product can be overwritten
+with this setting. Following product names can be set:
+
+* SLES/15/x86_64
+* SLES/15.1/x86_64
+
+[listing]
+migration_product: SLES/15/x86_64
+
+[NOTE]
+If the migration plugin is not used, the value set for the target
+product will not have any effect.
+
+Control Zypper Installation Mode::
+By default the upgrade system uses a plugin named: `migration`. This
+plugin sends a request to the SUSE Customer Center, asking for an upgrade
+path. The request response contains the list of repositories required to
+upgrade the system. This process only works for SLES registered systems.
+If the upgrade process is used on systems that are not registered or for
+which the SUSE Customer Center has no upgrade path, it's required to
+switch off the use of the migration plugin.
+
+[listing]
+use_zypper_migration_plugin: true|false
+
+[NOTE]
+If the migration plugin is not used, the upgrade process just calls
+`zypper dup` and expects all required repositories to be setup correctly.
+This means prior to the upgrade the setup of the repositories must
+be performed.
```

Comment:
I think, the order is a bit off. 

```
If the migration plugin is not used, the setup of the repositories must
be performed. Once done, the upgrade process just calls `zypper dup` and expects all required
repositories to be setup correctly.

Reply:
The point is that this repo setup has to be done prior running the migration, that's why the reason for this was mentioned first. But I follow your experience here and changed it :) Thanks